pom.xml

```diff
@@ -554,6 +554,11 @@ limitations under the License.
                     <artifactId>maven-clean-plugin</artifactId>
                     <version>3.1.0</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-enforcer-plugin</artifactId>
+                    <version>3.0.0-M2</version>
```

Comment:
any reason to go to a milestone release at 3.0.0-M2 rather than stick with the last stable version of 1.4.1? I'm not necessarily against it, but we tend to avoid non-final releases unless there is some good reason for upgrading.  Note that if we already use enforcer so you would probably want to delete the `<version>` defined here:

https://github.com/apache/tinkerpop/blob/8734301cfd692d43f911e29cd4b350beca19816f/pom.xml#L183

and allow `<pluginManagement>` to handle that.

Reply:
Yes, I will change it.